package.json

```diff
@@ -42,7 +42,7 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "0.65.x",
+        "branch": "c75bump-0.65.x",
```

Comment:
why the branch change here?

Reply:
This was for the CI. Removed the commit that made this change.